src/chrome/content/rules/Phillips_Academy_Andover.xml

```diff
@@ -17,13 +6,32 @@
 	<target host="colwizlive.andover.edu" />
 	<target host="sso.andover.edu" />
 	<target host="studentreports.andover.edu" />
-	<!--target host="mail.andover.edu" /-->
 	<target host="mypassword.andover.edu" />
 	<target host="student.andover.edu" />
 	<target host="patechmasters.com" />
 	<target host="*.patechmasters.com" />
 	
+	
+	<!-- andover.edu does not respond, but should be redirected to https://www.andover.edu to prevent MitM. -->
+	<target host="andover.edu" />
+	
+	<!-- mail.andover.edu returns an error, but should be included to protect mail.andover.edu/owa -->
+	<target host="mail.andover.edu" />
+	
+	<test url="http://andover.edu" />
```

Comment:
Both of the test urls here are duplicated with each other. Also, those test urls will be implicitly created by the corresponding `<target>` tag. Extra test URLs are only needed then the implicit test URL is not sufficient, e.g. when a domain has some problematic parts or needs a more complex URL.